sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -495,7 +487,7 @@ export class BatchingReceiverLite {
       };
 
       abortSignalCleanupFunction = checkAndRegisterWithAbortSignal((err) => {
-        cleanupBeforeResolveOrReject("removeDrainHandler");
+        cleanupBeforeResolveOrReject();
```

Comment:
[Unrelated to this PR]

What's the point of calling `cleanupBeforeResolveOrReject` inside `abortSignalCleanupFunction`?

Looks like `abortSignalCleanupFunction` is ever being called only in the `cleanupBeforeResolveOrReject`.